src/views/CartView.vue

```diff
@@ -1,8 +1,12 @@
 <template>
   <div id="cart-view" class="row">
     <div class="col">
-      <h3 class="mb-3">3. Order variables</h3>
+      <h3>{{ 'lifelines-webshop-cart-header' | i18n }}</h3>
       <template v-if="selectedVariableIds.length">
+        <div class="mb-3" v-if="selectedVariableIds.length > 0">
+          <button id="save" type="button" class="btn btn-primary" @click="save">Save</button>
```

Comment:
Maybe not a good idea to use a generic name like "save" as id. Usually better to use classes.